cats/cats-sql/src/main/kotlin/com/netflix/spinnaker/config/SqlCacheConfiguration.kt

```diff
@@ -124,9 +126,12 @@ class SqlCacheConfiguration {
       log.info("Configured coroutine context with newFixedThreadPoolContext of $poolSize threads")
     }
 
+    // stores dates like java.time.Instant as a String with the format "2017-04-06T16:29:44.947869272Z" instead of a Map
+    val mapper = ObjectMapper().registerModule(JavaTimeModule()).disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+
     return SqlNamedCacheFactory(
       jooq,
-      ObjectMapper(),
+      mapper,
```

Comment:
Why not use the object mapper that's already in the application context?

Reply:
I just wanted to avoid other changes pulled from that one that we may not know about into a working sql cache, but I see the point of standardizing on one object mapper config for easier maintenance.

If I read correctly, the mapper in the application context is built from these settings: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/config/CloudDriverConfig.groovy#L132 . I did a test with that one and the date format is also fixed. Do you think it's safe to use it?

Reply:
Checking in, @robzienert should we use the mapper from the application context?

Reply:
It should be changed. Having different object mappers isn't only inefficient, it can be the source of bugs if one object mapper serializes things while another, differently configured one, deserializes it.

Reply:
Change can be made here: https://github.com/spinnaker/clouddriver/blob/master/clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/config/CloudDriverConfig.groovy#L132